build.make

```diff
@@ -69,12 +69,17 @@ endif
 # toolchain.
 BUILD_PLATFORMS =
 
+# Add go ldflags using LDFLAGS at the time of compilation.
+IMPORTPATH_LDFLAGS ?= -X main.version=$(REV) 
+EXT_LDFLAGS ?= -extldflags "-static"
+LDFLAGS = 
+FULL_LDFLAGS = $(LDFLAGS) $(IMPORTPATH_LDFLAGS) $(EXT_LDFLAGS)
```

Comment:
This seems inconsistent: why are `IMPORTPATH_LDFLAGS` and `EXT_LDFLAGS` set with `?=` and the others with `=`? Was that intentional?

I think I prefer `=` because that then only allows overriding only via `make <VARNAME>=`. In particular, `LDFLAGS` might be something that a user has set in the environment when invoking make, without being aware that it affects the Go compilation - it's not likely, but I'd prefer to avoid surprises.